src/components/intro/style.scss

```diff
@@ -10,6 +11,7 @@
   font-family: 'intro-text';
   font-style: normal;
   font-weight: 500;
+  font-display: fallback;
```

Comment:
Does this do anything now we're inlining?

Reply:
The data URLs do take a few ms according to lighthouse. I guess `block` might better suit that?

Reply:
Yeah make it block. That covers browsers that might allow a swap (Edge, maybe?)